lib/BillTechLinksManager.php

```diff
@@ -248,7 +248,7 @@ public static function intToMoney($value)
 	}
 
 	public function cancelPaymentLinksIfManuallyDeletedLiability() {
-		$paymentLinkTokensToCancel = $this->getPaymentLinksToCancel();
+		$paymentLinkTokensToCancel = $this->getPaymentLinksToCancel() ? $this->getPaymentLinksToCancel() : array();
```

Comment:
Chyba razem wrzuciliśmy poprawkę na to samo :) Tutaj 2x strzelasz do bazy, proponowałbym poprawić.

Reply:
Słuszna uwaga chciałem żeby było "elegancko". Mam nadzieję, że teraz będzie lepiej.